README.md

```diff
@@ -29,7 +29,8 @@ There's also an example project. In the example, you can swipe horizontally and
 There are two options:
 
  1. Shimmer is available as `Shimmer` in [Cocoapods](http://cocoapods.org).
- 2. Manually add the files into your Xcode project. Slightly simpler, but updates are also manual.
+ 2. Shimmer is available as `github "facebook/Shimmer"` in [Carthage](https://github.com/Carthage/Carthage).
```

Comment:
I would avoid adding this here. I'm happy to merge pull requests to add them, but maintaining opaque Xcode projects isn't something I think we can guarantee to work for future releases.